src/unique-id.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Rendering of the blocks based on the display condition.
+ *
+ * @package Stackable
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'stackable_get_random_string' ) ) {
+	function stackable_get_random_string( $length ) {
```

Comment:
Instead of random string, can you change this function to be stackable_generate_unique_id without the need to pass a length anymore?